api/models.py

```diff
@@ -1,3 +1,20 @@
 from django.db import models
 
-# Create your models here.
+class Projects(models.Model):
+	PROJECT_STATUS = (
+		('open', 'Opened'),
```

Comment:
Could you change the values that are going into the DB to be uppercase, the way @livike did in her code https://github.com/livike/TrackCat/blob/createtasksmodel/api/models.py#L8

It is not a deal breaker, but I find it to be a nice way of differentiate between the two values in the tuple. And also, since this is taking a role of a constant, convention for constants says, that they are (almost) always written with uppercase letters.